vms/platformvm/txs/executor/import_test.go

```diff
@@ -137,7 +137,7 @@ func TestNewImportTx(t *testing.T) {
 				},
 			),
 			sourceKeys:  []*secp256k1.PrivateKey{sourceKey},
-			timestamp:   env.config.BanffTime,
```

Comment:
this does not make sense in the first place. We initialize the environment to to test Apricot fork